cmd/kubeadm/app/cmd/init.go

```diff
@@ -281,7 +281,7 @@ func (i *Init) Run(out io.Writer) error {
 
 	// Create the default node bootstrap token
 	tokenDescription := "The default bootstrap token generated by 'kubeadm init'."
-	if err := nodebootstraptokenphase.UpdateOrCreateToken(client, i.cfg.Token, false, i.cfg.TokenTTL, kubeadmconstants.DefaultTokenUsages, tokenDescription); err != nil {
+	if err := nodebootstraptokenphase.UpdateOrCreateToken(client, i.cfg.Token, false, i.cfg.TokenTTL, kubeadmconstants.DefaultTokenUsages, []string{}, tokenDescription); err != nil {
```

Comment:
> The default kubeadm token should have a prefix of system:bootstrappers:kubeadm:default-node or something.

That seems like you're using a group to represent an individual token... each token already has its own unique username... why not use that?